data_model/clusters/AccountLogin.xml

```diff
@@ -54,44 +54,59 @@ This notice and disclaimer must be included on all copies of this document.
 Connectivity Standards Alliance
 508 Second Street, Suite 206
 Davis, CA 95616, USA
--->
-<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" id="0x050e" name="Account Login" revision="1">
-  <revisionHistory>
-    <revision revision="1" summary="Initial Release"/>
-  </revisionHistory>
-  <classification hierarchy="base" role="application" picsCode="ALOGIN" scope="Endpoint"/>
-  <commands>
-    <command id="0x00" name="GetSetupPIN" response="GetSetupPINResponse">
-      <access invokePrivilege="admin" fabricScoped="true" timed="true"/>
-      <mandatoryConform/>
-      <field id="0" name="TempAccountIdentifier" type="string">
-        <mandatoryConform/>
-        <constraint type="lengthBetween" from="16" to="100"/>
-      </field>
-    </command>
-    <command id="0x01" name="GetSetupPINResponse" direction="responseFromServer">
-      <access invokePrivilege="operate" fabricScoped="true"/>
-      <mandatoryConform/>
-      <field id="0" name="SetupPIN" type="string">
-        <mandatoryConform/>
-        <constraint type="desc"/>
-      </field>
-    </command>
-    <command id="0x02" name="Login" response="Y">
-      <access invokePrivilege="admin" fabricScoped="true" timed="true"/>
-      <mandatoryConform/>
-      <field id="0" name="TempAccountIdentifier" type="string">
-        <mandatoryConform/>
-        <constraint type="lengthBetween" from="16" to="100"/>
-      </field>
-      <field id="1" name="SetupPIN" type="string">
-        <mandatoryConform/>
-        <constraint type="minLength" value="11"/>
-      </field>
-    </command>
-    <command id="0x03" name="Logout" response="Y">
-      <access invokePrivilege="operate" fabricScoped="true" timed="true"/>
-      <mandatoryConform/>
-    </command>
-  </commands>
+-->
+<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" id="0x050e" name="Account Login" revision="1">

```

Comment:
Why do these show updates? Is this a newline change (since I see no actual delta)?